zeppelin/1.11/README.md

```diff
@@ -0,0 +1,74 @@
+# How to use Apache Zeppelin on DC/OS
+
+[Apache Zeppelin](https://zeppelin.apache.org/) is a web-based notebook that enables you to do interactive data analytics. For example, you can use it as a front-end for [Apache Spark](https://github.com/dcos/examples/tree/master/1.8/spark).
+
+- Estimated time for completion: 8 minutes
+- Target audience: Data scientists and data engineers that want an interactive data analytics tool.
+- Scope: Install and use Zeppelin in DC/OS.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Zeppelin](#install-zeppelin)
+- [Use Zeppelin](#use-zeppelin)
+- [Uninstall Zeppelin](#uninstall-zeppelin)
+
+## Prerequisites
+
+- A running DC/OS 1.11 cluster with 2 agents (one private, one public) each with 2 CPUs and 2 GB of RAM available.
+- [DC/OS CLI](https://dcos.io/docs/1.9/usage/cli/install/) installed.
+- [Marathon-LB](https://dcos.io/docs/1.9/usage/service-discovery/marathon-lb/) installed.
```

Comment:
Same here, for 1.11 this should point to the service docs: https://docs.mesosphere.com/services/marathon-lb/